flink/flink.sh

```diff
@@ -158,6 +163,18 @@ taskmanager.network.numberOfBuffers: ${FLINK_NETWORK_NUM_BUFFERS}
 fs.hdfs.hadoopconf: ${HADOOP_CONF_DIR}
 EOF
 
+  if [[ "${MASTER_ADDITIONAL}" != "" ]]; then
+    zookeeper_nodes="$(grep '^server\.' /etc/zookeeper/conf/zoo.cfg | uniq | cut -d '=' -f 2 \
+    | cut -d ':' -f 1 | xargs -L1 echo | sed "s/$/:2181/g" | xargs echo | sed "s/ /,/g")"
+    cat << EOF >> ${FLINK_INSTALL_DIR}/conf/flink-conf.yaml
+high-availability: zookeeper
+high-availability.zookeeper.quorum: ${zookeeper_nodes}
+high-availability.zookeeper.storageDir: hdfs:///flink/recovery
+high-availability.zookeeper.path.root: /flink
+yarn.application-attempts: 10
```

Comment:
Why is this value 10, but `yarn.resourcemanager.am.max-attempts` property is recommended to be set to 4?